helix-rest/src/main/java/org/apache/helix/rest/clusterMaintenanceService/api/OperationAbstractClass.java

```diff
@@ -19,14 +19,22 @@
  * under the License.
  */
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.List;
 import java.util.Map;
 
+import org.apache.helix.PropertyType;
 import org.apache.helix.rest.clusterMaintenanceService.MaintenanceManagementInstanceInfo;
 import org.apache.helix.rest.common.datamodel.RestSnapShot;
 
 
 interface OperationAbstractClass {
+
+  List<PropertyType> PROPERTY_TYPE_LIST = new ArrayList<>(Arrays
```

Comment:
Where is this used? Can or should this just be a static array?

Reply:
This value will be read in clusterMaintenanceService take/freeInstance implementation. Advising what are the type of property needed when populating the snapshot. 
Different child class may change this value, so may not be a static final.